sources/packages/api/src/route-controllers/application/application.controller.ts

```diff
@@ -331,24 +334,34 @@ export class ApplicationController extends BaseController {
     const student = await this.studentService.getStudentByUserId(
       userToken.userId,
     );
-    const application =
-      await this.applicationService.getAssessmentByApplicationId(
+
+    // TODO: temporary code to allow the assessment to be retrieved using the application id.
+    const [originalAssessment] =
+      await this.assessmentService.getAssessmentsByApplicationId(
         applicationId,
-        student.id,
+        AssessmentTriggerType.OriginalAssessment,
       );
-    if (!application) {
+    // TODO: end of the temporary code that will be removed.
+
```

Comment:
extra line

Reply:
This todo is to show what is supposed to be temporary. I am not sure if it should be considered an invalid extra line. Please see below what it looks like.

![image](https://user-images.githubusercontent.com/61259237/159070648-4858c3b0-9f58-4291-a7d9-ca1764f6af89.png)